Jenkinsfile

```diff
@@ -14,6 +14,7 @@ pipeline {
   environment {
     DOCKER_HOST = "unix:///var/run/docker.sock"
     AWS_DEFAULT_REGION = "eu-west-1"
+    RUN_END_TO_END_ON_PR = "${params.runEndToEndTestsOnPR}"
```

Comment:
do we need to add the parameter in a `parameters { ... }` block?

and we need to add a conditional to check to decide whether to run the end-to-end stage depending on the value of the parameter